_stbt/control.py

```diff
@@ -557,6 +558,24 @@ def lirc_key_reader(cmd_iter, control_name):
             yield m.group('key')
 
 
+def lircd_remote_listen(lircd_socket=None):
```

Comment:
We want to be using the word "control" instead of "remote" in any new code. Perhaps "fake_lircd_listen"?